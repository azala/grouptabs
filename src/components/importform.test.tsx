import React from "react";
import { render } from "@testing-library/react";
import ImportForm from "./importform";

it("renders form", () => {
  const { container } = render(
    <ImportForm
      tabId="xy123hq"
      onTabIdChange={jest.fn()}
      onSubmit={jest.fn()}
    />
  );
  expect(container.firstChild).toMatchSnapshot();
});
