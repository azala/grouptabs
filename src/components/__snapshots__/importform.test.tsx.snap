// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders form 1`] = `
<form
  class="import-form"
>
  <div
    class="row-label"
  >
    Open shared tab:
  </div>
  <input
    class="full-width"
    placeholder="Tab ID â€¦"
    type="text"
    value="xy123hq"
  />
  <button>
    Open
  </button>
  <div
    class="error-message"
  />
</form>
`;
